BT-Tracking/Interface/Help/HelpVC.swift

```diff
@@ -34,8 +33,80 @@ final class HelpVC: MarkdownController {
 
         title = L10n.helpTitle
         navigationItem.rightBarButtonItem?.title = L10n.about
+        navigationController?.tabBarItem.title = L10n.helpTabTitle
```

Comment:
Neni to duplicita s radkem 27?

Reply:
Ono se to prepise tim title na radku 34, ale zaroven to pred viewDidLoad je potreba nastavit